src/sentry/static/sentry/app/actionCreators/indicator.tsx

```diff
@@ -30,33 +49,35 @@ export function addMessage(msg, type, options = {}) {
   IndicatorActions[action](msg, type, {...options, duration});
 }
 
-function addMessageWithType(type) {
-  return (msg, duration, options = {}) => addMessage(msg, type, {...options, duration});
+function addMessageWithType(type: IndicatorType) {
+  return (msg: React.ReactNode, duration?: number, options?: Omit<Options, 'duration'>) =>
+    addMessage(msg, type, {...options, duration});
 }
 
-export function addLoadingMessage(msg = t('Saving changes...'), ...args) {
+export function addLoadingMessage(
+  msg: React.ReactNode = t('Saving changes...'),
+  ...args
+) {
   return addMessageWithType('loading')(msg, ...args);
 }
 
-export function addErrorMessage(...args) {
-  return addMessageWithType('error')(...args);
+export function addErrorMessage(msg: React.ReactNode, ...args) {
+  return addMessageWithType('error')(msg, ...args);
 }
 
-export function addSuccessMessage(...args) {
-  return addMessageWithType('success')(...args);
+export function addSuccessMessage(msg: React.ReactNode, ...args) {
+  return addMessageWithType('success')(msg, ...args);
 }
 
-const PRETTY_VALUES = {
-  '': '<empty>',
-  [null]: '<none>',
-  [undefined]: '<unset>',
-  [false]: 'disabled',
-  [true]: 'enabled',
-};
+const PRETTY_VALUES: Map<unknown, string> = new Map([
```

Comment:
Ah I converted this to Map hoping TS would allow me to use boolean as a key, but alas no

Reply:
IMO I would just type the values in the `PRETTY_VALUES` as anys

```
const PRETTY_VALUES = {
  [false as any]: 'disabled',
  ...
}